time stamp
float64 batt_voltage
float64[2] cmd_rpm
float64[2] out_voltage
float64[2] msd_rpm
geometry_msgs/Twist msd_vel